src/org/thoughtcrime/securesms/ConversationListAdapter.java

```diff
@@ -195,7 +195,12 @@ interface ItemClickListener {
   }
 
   void changeData(@Nullable DcChatlist chatlist) {
-    dcChatlist = chatlist == null ? new DcChatlist(0, 0) : chatlist;
+    if (chatlist == null) {
+      dcChatlist =  new DcChatlist(0, 0);
+    } else {
+      dcChatlist = chatlist;
+      dcContext = DcHelper.getAccounts(context).getAccount(chatlist.getAccountId());
```

Comment:
that would be my suggestion:

```suggestion
    Context context = this.context.get();
    if (context == null) {
      return;
    }

    if (chatlist == null) {
      dcChatlist =  new DcChatlist(0, 0);
    } else {
      dcChatlist = chatlist;
      if (dcContext.getAccountId() != chatlist.getAccountId()) {
        dcContext = DcHelper.getAccounts(context).getAccount(chatlist.getAccountId());
      }
```